addons/sale/tests/test_onchange.py

```diff
@@ -48,26 +49,19 @@ def test_onchange_product_id(self):
         fp_tax_id = self.fiscal_position_tax_model.create(dict(position_id=fp_id.id,
                                                                tax_src_id=tax_include_id.id,
                                                                tax_dest_id=tax_exclude_id.id))
-        so_vals = {
-            'partner_id': partner_id.id,
-            'pricelist_id': pricelist.id,
-            'fiscal_position_id': fp_id.id,
-            'order_line': [
-                (0, 0, {
-                    'name': product_id.name,
-                    'product_id': product_id.id,
-                    'product_uom_qty': 1.0,
-                    'product_uom': uom_id.id,
-                    'price_unit': 121.0
-                })
-            ]
-        }
-
-        so = self.so_model.create(so_vals)
-
-        so_line = so.order_line[0]
-        so_line.product_id_change()
-        self.assertEquals(100, so_line.price_unit, "The included tax must be subtracted to the price")
+
+        sale_order = self.so_model.create(dict(partner_id=partner_id.id,
```

Comment:
Since this test suite aims to check the onchange, create SO using SSF seems a better idea.